konduit-serving-rpm/src/main/resources/etc/profile.d/konduit-serving-env.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# This file defines the environment variables needed to run the konduit serving admin command.
+
+export KONDUIT_SERVING_BASE_ARTIFACT_NAME=konduit-serving
+export KONDUIT_SERVING_DIRECTORY_STRUCTURE=konduit/serving
+export KONDUIT_SERVING_VERSION=0.1.0-SNAPSHOT
+export KONDUIT_SERVING_BACKEND=cpu
+
+export KONDUIT_SERVING_HOME=/opt/${KONDUIT_SERVING_DIRECTORY_STRUCTURE}
+export CONDA_INSTALL_PATH=${KONDUIT_SERVING_HOME}/miniconda
```

Comment:
We don't really have the miniconda concept unless this is going to be used for some form of automatic setting of the python path? @maxpumperla can you comment here?

Reply:
There's an idea about being able to set the python path automatically using the konduit cli for configurations for python steps. It would look something like this: 
`konduit update-python-path <yaml.config.file>

Reply:
@agibsonccc If I understand correctly, Shams wants to have an _optional_ conda installation for users that don't want to rely on their own Python install locally (or don't even have one). in the case of this rpm install it might actually be useful to just ship everything, batteries included. this way you can directly get started with the `konduit` cli without having to do any additional steps (like pip install konduit etc.)

Reply:
Including everything doesn't mean using conda. Why does it have to be conda?